apps/fz_http/lib/fz_http_web/controllers/json/device_controller.ex

```diff
@@ -0,0 +1,47 @@
+defmodule FzHttpWeb.JSON.DeviceController do
+  @moduledoc """
+  REST API Controller for Devices.
+  """
+
+  use FzHttpWeb, :controller
+
+  action_fallback FzHttpWeb.JSON.FallbackController
+
+  alias FzHttp.Devices
+
+  def index(conn, _params) do
+    # XXX: Add user-scoped devices
```

Comment:
Can you please clarify this TODO? In 2 months we won't be able to tell what it means

Reply:
Even better -- I removed it. I'm sure we'll have a list of API updates to make after user feedback.